### PR TITLE
Porting ClimberCalibrate, ClimberRun, ClimberSetGatehook, ClimberTimerOverride to Java

### PR DESCRIPTION
**Pull request by hpark-24**

### CODE REVIEW

**Review by amull23 — APPROVED** [review]
looks good!

File: Offseason2022/src/main/java/frc/robot/commands/ClimberSetGatehook.java [hunk 1]
```diff
@@ -4,22 +4,28 @@
...
 
 /**
  *
  */
 public class ClimberSetGatehook extends CommandBase
 {
-  public ClimberSetGatehook( )
+  private boolean m_climberGateHookClosed;
+
+  public ClimberSetGatehook(boolean climberSetGateHook)
```
Comment (jmullins16) [inline Offseason2022/src/main/java/frc/robot/commands/ClimberSetGatehook.java:16]: An improvement for readability in variable naming here. When ClimberSetGateHook() is being called, the caller is requesting that the gate hook be put in one of two states--open or closed. Naming the boolean parameter something like "gateHookClosed" and passing in true or false would be easier for the reader to understand. And change the m_ variable to match it.

File: Offseason2022/src/main/java/frc/robot/commands/ClimberTimerOverride.java [hunk 2]
```diff
@@ -3,23 +3,31 @@
...
 /**
  *
  */
 public class ClimberTimerOverride extends CommandBase
 {
-  public ClimberTimerOverride( )
+  private Climber        m_climber;
+  private XboxController operatorController;
+
+  public ClimberTimerOverride(Climber climber)
```
Comment (jmullins16) [inline Offseason2022/src/main/java/frc/robot/commands/ClimberTimerOverride.java:19]: We will be modifying all of our "simple" commands to pass (inject) all subsystems in as parameters. So in addition to the Climber subsystem, passing the second parameter as "XboxController gamepad" would be more direct than using RobotContainer.getInstance(). Try to add this second parameter and see if you can simplify the references to the operator controller.

**Review by jmullins16 — CHANGES_REQUESTED** [review]
Looks really good. Just a couple of small changes, and I'd like to get this merged soon.

**Review by jmullins16 — APPROVED** [review]
Looks good now! Ready to merge.

**Pull request merged**